raiden/tests/scenarios/scenario-funding channel.yaml

```diff
@@ -0,0 +1,256 @@
+            - deposit: {from: 0, to: 1, total_deposit: 79_000_000_000_000_000_000}
+            - deposit: {from: 0, to: 1, total_deposit: 80_000_000_000_000_000_000}
+            - deposit: {from: 0, to: 1, total_deposit: 81_000_000_000_000_000_000}
+            - deposit: {from: 0, to: 1, total_deposit: 82_000_000_000_000_000_000}
+            - deposit: {from: 0, to: 1, total_deposit: 83_000_000_000_000_000_000}
+            - deposit: {from: 0, to: 1, total_deposit: 84_000_000_000_000_000_000}
+            - deposit: {from: 0, to: 1, total_deposit: 85_000_000_000_000_000_000}
+            - deposit: {from: 0, to: 1, total_deposit: 86_000_000_000_000_000_000}
+            - deposit: {from: 0, to: 1, total_deposit: 87_000_000_000_000_000_000}
+            - deposit: {from: 0, to: 1, total_deposit: 88_000_000_000_000_000_000}
+            - deposit: {from: 0, to: 1, total_deposit: 89_000_000_000_000_000_000}
+            - deposit: {from: 0, to: 1, total_deposit: 90_000_000_000_000_000_000}
+            - deposit: {from: 0, to: 1, total_deposit: 91_000_000_000_000_000_000}
+            - deposit: {from: 0, to: 1, total_deposit: 92_000_000_000_000_000_000}
+            - deposit: {from: 0, to: 1, total_deposit: 93_000_000_000_000_000_000}
+            - deposit: {from: 0, to: 1, total_deposit: 94_000_000_000_000_000_000}
+            - deposit: {from: 0, to: 1, total_deposit: 95_000_000_000_000_000_000}
+            - deposit: {from: 0, to: 1, total_deposit: 96_000_000_000_000_000_000}
+            - deposit: {from: 0, to: 1, total_deposit: 97_000_000_000_000_000_000}
+            - deposit: {from: 0, to: 1, total_deposit: 98_000_000_000_000_000_000}
+            - deposit: {from: 0, to: 1, total_deposit: 99_000_000_000_000_000_000}
+            - assert: {from: 0, to: 1, total_deposit: 99_000_000_000_000_000_000, balance: 99_000_000_000_000_000_000, state: "opened"}
+
+      - serial:
+          name: Scenario F9 (Depositing 0 tokens)
+          tasks:
+            - deposit: {from: 0, to: 1, total_deposit: 99_000_000_000_000_000_000} ## we expect this to work, right?
+            - assert: {from: 0, to: 1, total_deposit: 99_000_000_000_000_000_000, balance: 99_000_000_000_000_000_000, state: "opened"}
+
+      - serial:
+          name: Scenario F10 (Depositing negative amount I)
+          tasks:
+            - deposit: {from: 0, to: 1, total_deposit: -1, expected_http_status: 409}
+            - assert: {from: 0, to: 1, total_deposit: 99_000_000_000_000_000_000, balance: 99_000_000_000_000_000_000, state: "opened"}
+
+      - serial:
+          name: Scenario F11 (Depositing negative amount II)
+          tasks:
+            - assert: {from: 0, to: 1, total_deposit: 99_000_000_000_000_000_000, balance: 99_000_000_000_000_000_000, state: "opened"} ## This seems to be a bug. When a node successfully deposits 98_000_000_000_000_000_000 (see code 200), it's status should not be 99_000_000_000_000_000_000
```

Comment:
Does the stuff describe in the commit still happen?

Reply:
this is solved now and we expect code 409